t/01-pdk/04-request/17-get_path_with_query.t

```diff
@@ -50,3 +52,53 @@ GET /t?foo=1&bar=2
 path_and_querystring=/t?foo=1&bar=2
 --- no_error_log
 [error]
+
+
```

Comment:
Please run the OpenResty `reindex` script on this file to properly format it. E.g. here we need 3 blank lines.

Reply:
Done!